gradle.properties

```diff
@@ -0,0 +1,3 @@
+ossrhUsername=
+ossrhPassword=
+
```

Comment:
Without that, `gradle build` / `gradle install` fails
I'm not a gradle expert but I think it should be builtable without any env variables set (which are used only for signing/publishing). Maybe there is a way to make gradle not to require those properties upfront?
Or maybe a comment in `README.MD` how to build OOB.